encord/orm/filter_preset.py

```diff
@@ -51,11 +55,9 @@ class CreatePresetParams(BaseDTO):
 
 class CreatePresetPayload(BaseDTO):
     name: str
-    description: Optional[str] = ""
```

Comment:
This is not a real parameter

Reply:
This is emblematic of Active/ Index being a bad overload of two very similiar functionalitys branching at random points. For some reason, we do have a description that we don't render